### PR TITLE
feat: introducing changelog flag for boot

### PR DESCRIPTION
**Pull request by rebelchris**
Added a new alert flag for lastChangelog, FE will be responsible for updating it through existing channels.
Added a new worker to set a redis cached value of our last changelog date.
Boot can then check wether this date is more up to date, and set Changelog to true.

WT-1054 #done

### CODE REVIEW

File: src/routes/boot.ts [hunk 1]
```diff
@@ -89,6 +98,7 @@ export default async function (fastify: FastifyInstance): Promise<void> {
         notifications: { unreadNotificationsCount },
         squads,
         features,
+        changelog: alerts.lastChangelog < new Date(lastChangelog),
```
Comment (idoshamun) [inline src/routes/boot.ts:101]: I forgot we have an alerts object. Should we put this inside the alerts object? Makes more sense I guess?

**Comment by idoshamun** [comment]
@rebelchris let me know when tests are green so I can have another look :)

File: src/routes/boot.ts (hunk 1 above)
Comment (rebelchris) [reply src/routes/boot.ts:101]: I added the actual changelog date as the user object.
The evaluation always needs to happen seperate, not sure it makes sense to sync that to the database?

Comment (idoshamun) [reply src/routes/boot.ts:101]: We don't need to sync that to the db. I mean something like:
```
alerts: {
  ...alerts,
  lastChangelog: alerts.lastChangelog < new Date(lastChangelog),
}
```
Client doesn't really need the timestamp itself just the true/false.

**Comment by idoshamun** [comment]
@rebelchris I don't remember but do we have a mutation to update the user's alert?

File: src/routes/boot.ts (hunk 1 above)
Comment (rebelchris) [reply src/routes/boot.ts:101]: Ah right, yeah we can do something like that.
The update will always happen as a date anyway.

Comment (rebelchris) [reply src/routes/boot.ts:101]: Hmm, but userAlerts is actually a datetime in the database, then we're mixing up types which might not be great.
Maybe we can simply return alerts.changelog seperate?

Comment (idoshamun) [reply src/routes/boot.ts]: Ok, np 😁

File: src/routes/boot.ts [hunk 2]
```diff
@@ -92,7 +104,7 @@ export default async function (fastify: FastifyInstance): Promise<void> {
       });
     }
     return res.send({
-      alerts: ALERTS_DEFAULT,
+      alerts: { ...ALERTS_DEFAULT, lastChangelog: false },
```
Comment (idoshamun) [inline src/routes/boot.ts:107]: This should be changelog I guess?

Comment (rebelchris) [reply src/routes/boot.ts:107]: Ah yes will change that 🙌

Comment (rebelchris) [reply src/routes/boot.ts]: Fixed

**Review by idoshamun — APPROVED** [review]

**Pull request merged**